### PR TITLE
CCTRI-1858: Fixed "Endpoint request timed out" error

### PR DESCRIPTION
**Pull request by MaxMhonc**

### CODE REVIEW

File: api/enrich.py [hunk 1]
```diff
@@ -38,13 +40,21 @@ def observe_observables():
     client = Client(key, url, headers=headers)
     limit = current_app.config['CTR_ENTITIES_LIMIT']
 
+    observables_ = []
     for observable in observables:
         observable = Observable.instance_for(**observable)
-        if observable is None:
-            continue
+        if observable is not None:
+            observables_.append(observable)
```
Comment (AlinaBlahun) [inline api/enrich.py:47]: Too many observable and observables here - hard to read. Please, rename something, ex. ```observables``` to ```input_observables ```

Comment (MaxMhonc) [reply api/enrich.py]: Done.

File: api/enrich.py [hunk 2]
```diff
@@ -38,12 +40,20 @@ def observe_observables():
...
+    prepared_observables = []
+    for input_observable in input_observables:
+        prepared_observable = Observable.instance_for(**input_observable)
+        if prepared_observable is not None:
+            prepared_observables.append(prepared_observable)
+
+    def make_bundle(observable):
+        return observable.observe(client, limit=limit)
+
+    workers_number = min((cpu_count() or 1) * 5, len(prepared_observables))
```
Comment (GeorgiyII) [inline api/enrich.py:52]: you can create function in utils which will give workers number, because you have already 2 same strings in the project

Comment (MaxMhonc) [reply api/enrich.py]: Oh, you're right, I missed it.(
Done.

**Review by GeorgiyII — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**